vassal-app/src/main/resources/VASSAL/i18n/Editor.properties

```diff
@@ -1930,6 +1930,7 @@ Editor.UsernameAndPasswordDialog.choose_your_weapons=Set your username and passw
 Editor.UsernameAndPasswordDialog.turn_your_key_sir=Please confirm your password
 
 # Validation Report Dialog
+Editor.ValidationReportDialog.blank_name=An invalid blank name found at
```

Comment:
I think this would read better as 

'Component must have a name at '.

To support translations in all languages, you need to include the object of the sentence as well. So this would be better as

'Component must have a name at %1$s'.

Then call is with

Resources.getString("Editor.ValidationReportDialog.blank_name", target.getClass().getName());